app/brave_settings_strings.grdp

```diff
@@ -326,13 +326,126 @@
+    Suggest another email alias
+  </message>
+  <message name="IDS_SETTINGS_EMAIL_ALIASES_NOTE_LABEL" desc="The label for the note">
+    Note
+  </message>
+  <message name="IDS_SETTINGS_EMAIL_ALIASES_EDIT_NOTE_PLACEHOLDER" desc="The placeholder for the edit note">
+    Enter a note for your address (optional)
+  </message>
+  <message name="IDS_SETTINGS_EMAIL_ALIASES_CANCEL_BUTTON" desc="The label for the cancel button">
+    Cancel
+  </message>
+  <message name="IDS_SETTINGS_EMAIL_ALIASES_MANAGE_BUTTON" desc="The label for the manage button">
+    Manage
+  </message>
+  <message name="IDS_SETTINGS_EMAIL_ALIASES_ALIAS_LABEL" desc="The label for the alias">
+    Email alias
+  </message>
+  <message name="IDS_SETTINGS_EMAIL_ALIASES_EMAILS_WILL_BE_FORWARDED_TO" desc="The label explaining where emails will be forwarded">
+    Emails will be forwarded to <ph name="MAIN_EMAIL">$1<ex>address@example.com</ex></ph>
+  </message>
+  <message name="IDS_SETTINGS_EMAIL_ALIASES_EDIT_ALIAS_TITLE" desc="The title for the edit alias">
+    Edit email alias
+  </message>
+  <message name="IDS_SETTINGS_EMAIL_ALIASES_CREATE_ALIAS_BUTTON" desc="The label for the create alias button">
+    Create
+  </message>
+  <message name="IDS_SETTINGS_EMAIL_ALIASES_SAVE_ALIAS_BUTTON" desc="The label for the save alias button">
+    Save
+  </message>
+  <message name="IDS_SETTINGS_EMAIL_ALIASES_SIGN_IN_OR_CREATE_ACCOUNT" desc="The label for the sign in or create account">
+    To get started, sign in or create a Brave account
+  </message>
+  <message name="IDS_SETTINGS_EMAIL_ALIASES_ENTER_EMAIL_TO_GET_LOGIN_LINK" desc="The label for the enter email to get login link">
+    Enter your email address to get a secure login link sent to your email. Clicking this link will either create or access a Brave Account and let you use the free Email Aliases service.
+  </message>
+  <message name="IDS_SETTINGS_EMAIL_ALIASES_GET_LOGIN_LINK_BUTTON" desc="The label for the get login link button">
+    Get login link
+  </message>
+  <message name="IDS_SETTINGS_EMAIL_ALIASES_EMAIL_ADDRESS_PLACEHOLDER" desc="The placeholder for the email address">
+    Email address
+  </message>
+  <message name="IDS_SETTINGS_EMAIL_ALIASES_LOGIN_EMAIL_ON_THE_WAY" desc="The label for the login email on the way">
+    A login email is on the way to <ph name="MAIN_EMAIL">$1<ex>address@example.com</ex></ph>
+  </message>
+  <message name="IDS_SETTINGS_EMAIL_ALIASES_CLICK_ON_SECURE_LOGIN" desc="The label for the click on secure login">
+    Click on the secure login link in the email to access your account.
+  </message>
+  <message name="IDS_SETTINGS_EMAIL_ALIASES_DONT_SEE_EMAIL" desc="The label for the don't see email">
+    Don't see the email? Check your spam folder or
+  </message>
+  <message name="IDS_SETTINGS_EMAIL_ALIASES_TRY_AGAIN" desc="The label for the try again">
+    try again.
+  </message>
```

Comment:
Not sure what this looks like in the UI, but typically it's not great for l10n to split mid-sentence. If the `try again` part is a link then it's better to use placeholders (`<ph name=...`).

Reply:
fixed